buildenv/jenkins/JenkinsfileBase

```diff
@@ -773,7 +777,21 @@ def addGrinderLink() {
 def run_parallel_tests() {
 	if (params.PARALLEL && params.PARALLEL != "None") {
 		stage ("Parallel Tests") {
-			parallel parallel_tests
+			def childJobs = parallel parallel_tests
+			node("$LABEL") {	
+				childJobs.each {
+					cjob ->
+					def jobInvocation = cjob.value.getRawBuild()
+					def buildId = jobInvocation.getNumber()
+					def name = cjob.value.getProjectName()
+					try {
+						copyArtifacts (projectName: "${name}", selector: specific("${buildId}"), filter: "**/TestTargetResult.tap", target:"${name}/${buildId}")
+					} catch (Exception e) {
+						echo "Cannot run copyArtifacts from ${name} with buildid ${buildId} . Skipping copyArtifacts..."
```

Comment:
Could we change the error msg to be more clear? For example, `Cannot copy TestTargetResult.tap from Build ${name} #${buildId} . Skipping copyArtifacts..."`